control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -442,24 +495,33 @@ func TestRun_TokensReplicatedDC(t *testing.T) {
 			responseCode := cmd.Run(cmdArgs)
 			require.Equal(t, 0, responseCode, ui.ErrorWriter.String())
 
+			// Sometimes it takes a moment for the policies to be settled when there
+			// is no kube secret being created.
+			sec.WaitForServiceIntentions(t)
```

Comment:
TODO: Figure out why we need this, it seems like `ACL().PolicyCreate()` is not strongly consistent enough for this to pass when we dont also have the added delays caused by provisioning the token+kube secret

Reply:
I would love to back out this change because it seems like unnecessary plumbing to expose the primary/secondary servers in each replicated setup test!